WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteTagsViewController.swift

```diff
@@ -101,13 +108,15 @@ final class SiteTagsViewController: UITableViewController {
         tableView.tableFooterView = UIView(frame: .zero)
         let nibName = UINib(nibName: TableConstants.cellIdentifier, bundle: nil)
         tableView.register(nibName, forCellReuseIdentifier: TableConstants.cellIdentifier)
-        setupRefreshControl()
+        activateRefreshControl()
     }
 
-    private func setupRefreshControl() {
-        let control = UIRefreshControl()
-        control.addTarget(self, action: #selector(refreshTags), for: .valueChanged)
-        refreshControl = control
+    private func activateRefreshControl() {
```

Comment:
I don't really like using a lazy load variable to initialize another variable, it's a bit hard to follow. We could do something like `setupRefreshControl` and `deactivateRefreshControl` and setup can initialize the variable if it's nil, what do you think?

Reply:
@elibud Works for me!